### PR TITLE
added more information about businesses and edited text

### PR DESCRIPTION
**Pull request by swiffcat**

### CODE REVIEW

File: running_a_business/oregon_saves.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+If you’re a business owner with at least one employee, you are required to register for OregonSaves by July 31.
+
+## How to register:
+
+* If you are new to OregonSaves, and wish to register your business, please call 1-844-661-1256 to receive an Access Code.
+
+* You can then use this code and your EIN to register.
+
+See the link below for more info and to register:
+https://www.oregonsaves.com/
```
Comment (ForzenAcademy) [inline running_a_business/oregon_saves.md]: You didn't make this a link

File: running_a_business/yearly_renewal.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Every year you must renew your LLC with the state. This must be done by the day you originally started your LLC. The cost is $100.
+
+## When It's Time to Ren​ew Your Business
+
+Renewal notices are mailed approximately 45 days before the renewal due date. You need to pay your fees and update your information by the renewal due date.
+
+## File Your Annual Re​port/Renewal and Pay Online
+Follow these steps to ren​ew​​ online at the following link:
+
+https://sos.oregon.gov/business/pages/renewal-help.aspx
```
Comment (ForzenAcademy) [inline running_a_business/yearly_renewal.md:12]: You didn't make this a link

File: running_a_business/oregon_saves.md (hunk 1 above)
Comment (swiffcat) [reply running_a_business/oregon_saves.md]: fixed

File: running_a_business/yearly_renewal.md (hunk 2 above)
Comment (swiffcat) [reply running_a_business/yearly_renewal.md:12]: fixed

**Review by ForzenAcademy — APPROVED** [review]

**Pull request merged**